docs/examples/sequence_processing/index.rst

```diff
@@ -0,0 +1,9 @@
+Sequence Processing
+===================
+
+.. toctree::
+   :maxdepth: 2
+
+   video/video_reader_simple_example.ipynb
```

Comment:
I would rather put it into data loading. As not processing is done in this video examples.

Reply:
That's why I wanted to put `data_loading` in `image` category. When using DALI you need to know how to load your data properly for your particular use-case...

I's leave it here, thinking about a guy that wants to now, how to process video in DALI. He'd go to `video` first, not `general` or any other stuff...

Reply:
If he would go to video why it is inside `Sequence Processing`?